docs/asciidoc/images/flux.svg

```diff
@@ -0,0 +1 @@
+../../../reactor-core/src/main/java/reactor/core/publisher/doc-files/marbles/flux.svg
```

Comment:
would it be possible to avoid the symlink trick with the `resources { }` instruction? like below snippet (from #2520):
```groovy
		resources {
			from("${sourceDir}/highlight") {
				into "./highlight/"
				include '**/*'
			}
			from("${sourceDir}/images") {
				into "./images/"
				include '**'
			}
			from( "${rootDir}/docs/svg" ) {
				into "./images/"
				include 'legend-*.svg'
			}
		}
```

Reply:
Could for the final rendition, but online preview (eg in IDE) would not see the image. Given that those two would be the only two where this happens, could be a bit puzzling for users...

Reply:
I don't think that is a terrible issue for the asciidoc, since it is intended to be consumed in HTML form, whereas the javadocs are also very very often consumed inline in the IDE.

Reply:
Sure. OTOH I don't think using symlinks is problematic, is it?

Reply:
principle of least surprise from a maintenance perspective, we don't have any symlink anywhere else and I'd prefer to keep it that way if there is no absolute need for it